ibmcloud/terraform/common/variables.tf

```diff
@@ -0,0 +1,29 @@
+
+variable "ibmcloud_api_key" {}
```

Comment:
Should we consider marking this as sensitive: https://www.terraform.io/language/values/variables#suppressing-values-in-cli-output so a user's api_key doesn't get printed out?

Reply:
it seems that we follow the same examples style as https://github.com/IBM-Cloud/terraform-provider-ibm/search?q=ibmcloud_api_key and check related logs, the `ibmcloud_api_key` did't be printed out.

Reply:
Great, thanks for sharing that link. Looks like it is all good already then!